raiden_contracts/tests/test_channel_unlock.py

```diff
@@ -0,0 +1,178 @@
+from functools import reduce
+from raiden_contracts.utils.config import E_CHANNEL_UNLOCKED
+from ..utils.events import check_channel_unlocked
```

Comment:
both here and below you should not have relative imports.

You should be be able to do `from raiden_contracts.utils.events import check_channel_unlocked`.

In general please try to avoid relative imports unless in the same directory